src/controllers/portfolio/portfolio.ts

```diff
@@ -71,13 +71,7 @@ export class PortfolioController extends EventEmitter {
 
   validTokens: any = { erc20: {}, erc721: {} }
```

Comment:
What's the purpose of "any" here? This could get extracted in a separate type that can be reused for the extension too?

```
type ValidTokens = { erc20: { [key: string]: boolean }; erc721: { [key: string]: boolean } }
```

and then here:

```
validTokens?: ValidTokens = { erc20: {}, erc721: {} }
```

Reply:
PS: I know that's not part of your changes, but since it's near I mentioned it :)